package.json

```diff
@@ -28,7 +29,7 @@
     "pinia-plugin-persistedstate": "^3.2.1",
     "sass": "^1.75.0",
     "sass-loader": "^14.2.1",
-    "v-calendar": "^3.1.2",
+    "v-calendar": "^3.0.0",
```

Comment:
Did they remove the date range functionality in newer versions or something?

Reply:
I'm looking through my history to try to find the github issue but basically the bug didn't take `max-date` into consideration